script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9B.txt

```diff
@@ -46,94 +46,94 @@ The man concentrates the cold waves flooding out of himself into a single point,
+You would freeze to death in 2 seconds if you got within ten meters of him.//10メートル以内に近づけば２秒で凍死。
+At this rate, I'll eventually fall as well. It's no different to how people struck by disaster in the snow-capped mountains eventually succumb to sleep. When a human's body temperature is below 34℃ or above 43℃, their brain cells shut down and they fall unconscious.//雪山で遭難した人間が眠ってしまうのと同じだ。人の場合、体温が摂氏34度以下、あるいは摂氏43度以上で脳細胞が停止し、意識が消失してしまう。
 <Page18>
-I observe his entire surroundings and not just the individual.
-Without interrupting my thoughts, I let the knife slide automatically the moment I see it.
+Rather than focusing only on Vlov, I observe his surroundings as well.
+Without interrupting my thoughts, I let the knife slide automatically the moment I see something.
 <Page19>
 ―――During the process, I revise the conjecture that I had made earlier.
 That cold wave is spreading in all directions.
 But is the density really as uniform as I thought?
 <Page20>
-The fact that the floor is the coldest means that there is a difference in temperature even within the cold wave. There's the matter of whether the wave is wholly strong or weak, but more importantly, there is a weakness.
+The fact that the floor is the coldest area means that there is a difference in temperature even within the cold wave. There's still the matter of whether the wave is wholly strong or weak, but importantly, it's not perfect.//　地表が最も低温だという事は、寒波の中にも温度差はある。強いか弱いかの基準ではあるが欠点は存在する。
 It may protect him from all directions, but that doesn't mean it makes him invincible all around―――
 <Page21>
 "...So you protect your master on the brink of death. How brave, but futile."
-The long lance that struck down Arcueid gets picked up from the shadows once again.
-Perhaps that was him being spurred on, because behind him, a remarkably large ice tree sprouts branches―――
+The long lance that struck down Arcueid is drawn from the shadows once again.
+Perhaps he was just warming up before, because behind him, a remarkably large ice tree sprouts branches―――//'Priming the pump' それが呼び水になったのか、ヤツの背後にひときわ大きな樹氷が枝を張り―――
 "―――Now. I will release, in great numbers...!"
 <Page22>
-I break my train of thought and pour my focus into intercepting his attack.//text overflow, needs rephrasing
+I put my thoughts on hold and pour all of my attention into intercepting his attack.//Do not touch this page without playtesting
 This is impossible. I can't stop them all.
-There is always a missile coming from somewhere. ―――I won't allow them near me.
+I'll surely be hit somewhere. ―――No.//必ずどこかに被弾する。―――許されない。
 I can't be so naive.
 I can't even let them graze me.
 If I bleed out, it's over. The cold air will not spare even a single minute before coming for my life.
 That's why I must kill. That's why I must struggle.
-I clear my mind and look down only at the 'line'.
+I clear my mind and look only at the 'lines'.//俯瞰 but not sure how to phrase looking only at the lines while also commanding a large-scale view
```

Comment:
This line makes me picture like imagining like seeing Vlov in wireframe or something.
>意識を真っ新にして、ただ“線”だけを俯瞰する。
I clear my mind and construct an image only out of the 'lines'.